scripts/write_envs_to_file.sh

```diff
@@ -14,7 +14,3 @@ done
 for ENVVAR in ${BACKEND_ENV_VARS}; do
     echo "${ENVVAR}=${!ENVVAR}" >> backend.prod.env
 done
-
-for ENVVAR in ${BACKEND_ENV_VARS}; do
-    echo "${ENVVAR}=${!ENVVAR}" >> backend.dev.env
-done
```

Comment:
You should revert both commits, I didn't see a change at `docker-compose.dev.yml`

Reply:
`docker-compose.dev.yml` was deleted, it should no longer be there
`docker-compose.yml` contains no reference to backend.dev.env
`docker-compose.override.yml` references backend.dev.env but is not used in travis context

Reply:
Ok, then it's fine